crates/provider-sqldb-postgres/Cargo.toml

```diff
@@ -11,6 +11,9 @@ edition.workspace = true
 license.workspace = true
 repository.workspace = true
 
+[lib]
+doctest = false
+
```

Comment:
Re-reading the [Cargo documentation on this field](https://doc.rust-lang.org/cargo/reference/cargo-targets.html#the-doctest-field), it looks like it only applies to `cargo test`, but not to `cargo test --doc`, which explicitly runs doc tests.

So that being the case, it looks like the next best option for not running `doctests` in this crate without changing the `.wit` would be using the above `#![cfg(not(doctest))]` directive.

Reply:
@rvolosatovs feel free to resolve this thread either way you end up going, whether customizing the `wit` to include `ignore` in the comment blocks, or if you end up implementing `#![cfg(not(doctest))]` instead.